stdlib/3/multiprocessing/connection.pyi

```diff
@@ -1,36 +1,61 @@
-from typing import Any, Iterable, List, Optional, Tuple, Type, Union
 import socket
 import types
+from multiprocessing.connection import Listener
```

Comment:
This line is what causes CI to fail. We are in the `multiprocessing.connection` module, after all.